tests/migration_test.go

```diff
@@ -934,7 +934,10 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 					mount /dev/$(lsblk --nodeps -no name,serial | grep %s | cut -f1 -d' ') /mnt/servacc
 				`, secretDiskSerial)
 				userData := fmt.Sprintf("%s\n%s", tests.GetGuestAgentUserData(), mountSvcAccCommands)
-				tests.AddUserData(vmi, "cloud-init", userData)
+				networkData, err := libnet.CreateDefaultCloudInitNetworkData()
```

Comment:
Please update the title of the commit message.
> ipv6, tests: configure stateful dhcpv6 on fedora VMs via cloud init
The commit is not configuring stateful dhcpv6 on fedora VMs. `libnet.CreateDefaultCloudInitNetworkData` still has static ipv6 config.